### ISSUE TITLE
Update Tooltip Docs (to add info on non-interactive elements)

### ISSUE THREAD

**Issue by jake-costa** [jpmorganchase/salt-ds#5216]
### Related page(s)

https://www.saltdesignsystem.com

### Kind of issue

Missing information

### Description

We need to update our tooltip documentation to provide guidance on the interactivity of the Tooltip's anchor to address questions and use cases where devs are trying to add a tooltip to an non-interactive element.

Information that needs to be added:
Clarification on a Tooltip's anchor and when to use
Alternatives 

### Browser

- [x] Chrome
- [x] Safari
- [x] Firefox
- [ ] Microsoft Edge

### Operating system

- [x] macOS
- [x] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Are you a JPMorgan Chase & Co. employee?

- [x] I am an employee of JPMorgan Chase & Co.
